improve cen usecase, handler grpc proto response, improve test in sdk…

athiwatp opened the pull request:
… feature price update

athiwatp committed:
improve cen usecase, handler grpc proto response, improve test in sdk feature price update

Kafakk commented on cmd/gvel/layers/logic/rebalance_test.go:
```diff
@@ -6,9 +6,9 @@ import (
-	vclient "gitlab.com/velo-labs/cen/libs/client"
-	"gitlab.com/velo-labs/cen/libs/txnbuild"
+	"github.com/velo-protocol/DRSv1/cmd/gvel/entity"
+	vclient "github.com/velo-protocol/DRSv1/libs/client"
```
```suggestion
 "github.com/velo-protocol/DRSv1/libs/client"
```

Kafakk commented on cmd/gvel/layers/logic/mint_credit_test.go:
```diff
@@ -6,9 +6,9 @@ import (
-	vclient "gitlab.com/velo-labs/cen/libs/client"
-	"gitlab.com/velo-labs/cen/libs/txnbuild"
+	"github.com/velo-protocol/DRSv1/cmd/gvel/entity"
+	vclient "github.com/velo-protocol/DRSv1/libs/client"
```
```suggestion
	 "github.com/velo-protocol/DRSv1/libs/client"
```

Kafakk commented on testkit/get_collateral_health_check/main.go:
```diff
@@ -2,8 +2,8 @@ package main
 	"context"
-	cenGrpc "gitlab.com/velo-labs/cen/grpc"
-	"gitlab.com/velo-labs/cen/libs/client"
+	cenGrpc "github.com/velo-protocol/DRSv1/grpc"
```
```suggestion
	spec "github.com/velo-protocol/DRSv1/grpc"
```

Kafakk commented on node/app/entities/update_price.go:
```diff
@@ -0,0 +1,8 @@
+	SignedStellarTxXdr          string
+	Asset                       string
+	Currency                    string
+	PriceInCurrencyPerAssetUnit string
```
```suggestion
	PriceInCurrencyPerAssetUnit decimal
```

Kafakk commented on node/app/layers/deliveries/grpc/price_update_operation.go:
```diff
@@ -17,13 +17,18 @@ func (handler *handler) handlePriceUpdateOperation(ctx context.Context, veloTx *
+		PriceUpdateOpResponse: &spec.PriceUpdateOpResponse{
+			Asset:                       priceUpdateOutput.Asset,
+			Currency:                    priceUpdateOutput.Currency,
+			PriceInCurrencyPerAssetUnit: priceUpdateOutput.PriceInCurrencyPerAssetUnit,
```
```suggestion
			PriceInCurrencyPerAssetUnit: priceUpdateOutput.PriceInCurrencyPerAssetUnitTruncate(7).StringFixed(7),
```

Kafakk committed:
Resolve
 - import package name
 - change data type of `PriceInCurrencyPerAssetUnit` to decimal in `UpdatePriceOutput` struct

Kafakk committed:
rename import name `spec` to `cenGrpc`
